### ISSUE TITLE
1.36.0-insider macOS cannot new and close active editor by key command + n, command + w

### ISSUE THREAD

**Issue by yamachu** [microsoft/vscode#75678]
- VSCode Version:
1.36.0-insider (7f53e50cbf316dff94fbbe5406d8f352dac99a6b)
- OS Version:
macOS 10.14.5（18F132）

Steps to Reproduce:

1. Open editor
2. Push command + n or command + w

Does this issue occur when all extensions are disabled?: Yes/No
Yes

---

developer tools shows

```
keybindingService.ts:370 Ctrl/Cmd+Num keybindings should not be used by default in web. Offender:  000141;000121  for  editor.foldAll
_assertBrowserConflicts @ keybindingService.ts:370
keybindingService.ts:370 Ctrl/Cmd+Num keybindings should not be used by default in web. Offender:  000141;000129  for  editor.foldAllMarkerRegions
...
keybindingService.ts:346 Ctrl/Cmd+W keybindings should not be used by default in web. Offender:  000153  for  workbench.action.closeActiveEditor
```

**Comment by octref** [comment]
@rebornix Seems you made a lot of changes in that area yesterday.
This was working until today's update for me.

**Comment by Prkns** [comment]
Just did the most recent update myself and can vouch for this issue. Had to re-add the key shortcuts back manually (which doesn't cause any conflicts) but just wondering why they'd been removed to begin with?

**Comment by adrianpacala** [comment]
Until this issue gets fixed, you can re-add those shortcuts manually with `[⌘K ⌘S]`:

```json
[
  {
    "command": "workbench.action.closeActiveEditor",
    "key": "cmd+w"
  },
  {
    "command": "workbench.action.files.newUntitledFile",
    "key": "cmd+n"
  }
]
```

**Comment by anilanar** [comment]
It's more than that, many keybindings are missing. Another example is `cmd+t` for go to symbol in ws.

**Comment by Prkns** [comment]
There's a few for sure, but the same temporary fix applies.

Whichever shortcuts you're missing at the moment, just manually add them back till it's either permanently fixed or an explanation is given 👍🏻

There's probably some missing that you'll never use, so they're no big deal to leave alone for the time being.

**Comment by yamachu** [comment]
This issue has fixed in latest release (68a7e5bc437b38d0281df0756997a25da2a2900c).
Thanks!
